websrc/gadget_menu/src/gadget_menu_main.tsx

```diff
@@ -37,15 +37,17 @@ interface GadgetInfoPanelProps
 {
 	manifest: AardvarkManifest;
 	highlight?: GadgetSeedHighlight;
+	autoLaunchSet?: boolean;
 	deleteCallback?: () => void;
+	autoLaunchCallBack?: () => void;
```

Comment:
Is this a toggle? If so, it should have toggle in its name.

Reply:
it is! and now does